docs/development/api/modules/plugin_registry.md

```diff
@@ -639,7 +639,7 @@ registerResourceTableColumnsProcessor(function ageRemover({ id, columns }) {
   if (id === 'headlamp-pods') {
     columns.push({
       label: 'Init Containers',
-      getter: (pod: Pod) => {
+      getValue: (pod: Pod) => {
```

Comment:
This is a better name, but is getter also available for backwards compatibility?

Reply:
yes getter is still available and works but marked as deprecated, there was an idea to catch only places where getter returns JSX but it wasn't reliable so a new field getValue was added instead. you can look at [ResourceTable.tsx:54](https://github.com/headlamp-k8s/headlamp/blob/81b5dd8063bf74e2344a3a1d686548c319665206/frontend/src/components/common/Resource/ResourceTable.tsx#L54) for more info

Reply:
Is getValue required now for ordering or something else? Or will getter work as before for everything?

Reply:
getValue is there for filtering, sorting the column and expects a plaintext value, render is a new property that controls rendering, getter will work as before but filtering or sorting won't work